great_expectations/data_context/templates.py

```diff
@@ -22,51 +18,69 @@
 
 # Datasources tell Great Expectations where your data lives and how to get it.
 # You can use the CLI command `great_expectations add-datasource` to help you
-# add a new datasource.
-#
-# Read more here at https://docs.greatexpectations.io/en/""" + rtd_url_ge_version + """/features/datasource.html
+# add a new datasource. Read more at https://docs.greatexpectations.io/en/""" + rtd_url_ge_version + """/features/datasource.html
 datasources: {}
-
 """
 
 CONFIG_VARIABLES_INTRO = """
-# This config file supports variable substitution which enables two use cases:
-#   - Secrets are kept out of committed files.
-#   - Configuration parameters can change based on the environment. For example:
-#     dev vs staging vs prod.
+# This config file supports variable substitution which enables: 1) keeping
+# secrets out of source control & 2) environment-based configuration changes
+# such as staging vs prod.
 #
-# When GE encounters substitution syntax (like the examples below) in the config 
-# file it will attempt to replace the value of “my_key” with the value from an 
-# environment variable “my_value” or a corresponding key read from the file
-# specified using `config_variables_file_path`. An environment variable will
-# always take precedence.
-#   - `my_key: ${my_value}
-#   - `my_key: $my_value`
+# When GE encounters substitution syntax (like `my_key: ${my_value}` or 
+# `my_key: $my_value`) in the config file it will attempt to replace the value
+# of `my_key` with the value from an environment variable `my_value` or a
+# corresponding key read from the file specified using
+# `config_variables_file_path`. Environment variables take precedence.
 #
 # If the substitution value comes from the config variables file, it can be a
 # simple (non-nested) value or a nested value such as a dictionary. If it comes
-# from an environment variable, it must be a simple value.
-"""
+# from an environment variable, it must be a simple value. Read more at:
+# https://docs.greatexpectations.io/en/""" + rtd_url_ge_version + """/reference/data_context_reference.html#managing-environment-and-secrets"""
 
 PROJECT_OPTIONAL_CONFIG_COMMENT = CONFIG_VARIABLES_INTRO + """
 config_variables_file_path: uncommitted/config_variables.yml
 
 # The plugins_directory will be added to your python path for custom modules
 # used to override and extend Great Expectations.
-
 plugins_directory: plugins/
 
-# Stores are configurable places to store things like expectations, evaluation
-# results, data docs, and more.
-# 
-# Three stores are required: expectations, profiling, and evaluation parameters,
-# and must be named here. Additional stores can be configured below and used for
-# data_docs, validation_operators or other purposes.
-#
-# Point the following required store names to the appropriate `stores` entry.
+# Validation Operators are customizable workflows that do things after you 
```

Comment:
Validation Operators perform validation, not just the post-validation actions.

I suggest we replace this sentence with:
"Validation Operators are customizable workflows for both bundling the validation of multiple expectation suites and the actions that should be taken after the validation."

Reply:
Excellent. Fixed.